### PR TITLE
Fix up shell scripts

### PR DESCRIPTION
**Pull request by jimberlage**
This brings these two scripts in line with the style guidelines.

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [ ] There are tests for these changes OR
- [X] These changes do not require tests because it's a small change to bring some shell scripts in line with the style guide

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/12299)

### CODE REVIEW

**Comment by jimberlage** [comment]
This should get #12295 to pass.

**Comment by emilio** [comment]
@bors-servo: r+

Thanks! You could also have done this in #12295 directly though :)

File: ports/geckolib/gecko_bindings/tools/setup_bindgen.sh [hunk 1]
```diff
@@ -12,7 +20,8 @@ fi
 
 # Make sure we have llvm38.
 if [ ! -x "$(command -v clang-3.8)" ]; then
-    echo "llmv38 must be installed. Mac users should |brew install llvm38|, Linux varies by distro."
+    echo "llmv38 must be installed." \
```
Comment (waddlesplash) [inline ports/geckolib/gecko_bindings/tools/setup_bindgen.sh]: `llmv38`!

**Comment by larsbergstrom** [comment]
@bors-servo retry
- #12314

**Pull request merged**